processing/src/main/java/org/apache/druid/java/util/metrics/OshiSysMonitorConfig.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.java.util.metrics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableSet;
+
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class OshiSysMonitorConfig
+{
+  public static final String PREFIX = "druid.monitoring.oshisys";
```

Comment:
there is no need to have `oshi` in there. OshiSysMonitor is the default sys monitor now. We will likely get rid of SysMonitor completely and make that a symlink to OshiSysMonitor. We will not be able to change these prefixes later.

Reply:
But, As of this commit what do you suggest naming it ?

Reply:
`druid.monitoring.sys`

> Because , we don't want to update the docs later ?
Because people would have already set the config and we can't change it without breaking their setup.

Reply:
If this is universally the case with sys metrics emitted in the containerized environments, should Druid automatically set the useful categories in the `K8sTaskAdapter` while launching the task in the container?

Reply:
> druid.monitoring.sys

done.

Regarding `K8sTaskAdapter` I think it depends on the type of installation and requirements of user to specify which categories to specify and hence I would not enforce anything.